From 34c6007dc8c5e4e4dce453ba617c8ec922155152 Mon Sep 17 00:00:00 2001
From: phanirithvij <phanirithvij2000@gmail.com>
Date: Tue, 30 Jun 2026 06:37:34 +0530
Subject: [PATCH 2/2] try google

Signed-off-by: phanirithvij <phanirithvij2000@gmail.com>
---
 fetcher/fetcher.go | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/fetcher/fetcher.go b/fetcher/fetcher.go
index 9c20baa..2cc5f41 100644
--- a/fetcher/fetcher.go
+++ b/fetcher/fetcher.go
@@ -10,7 +10,8 @@ import (
 
 /* List of Maven repo URLs to try when searching for POMs */
 var DefaultRepos = []string{
-	"https://repo.maven.apache.org/maven2",
+	//"https://repo.maven.apache.org/maven2",
+	"https://maven-central.storage-download.googleapis.com/maven2",
 	"https://dl.google.com/dl/android/maven2",
 	"https://plugins.gradle.org/m2",
 	"https://jitpack.io",
-- 
2.54.0

